common/supplemental/likelySubtags.xml

```diff
@@ -2724,6 +2724,16 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="und_Zanb" to="cmg_Zanb_MN"/>
 		<!--{ ?; Zanabazar Square; ? } => { Classical Mongolian; Zanabazar Square; Mongolia }-->
 
+		<!-- START: Added by Unicode 16, temporary -->
```

Comment:
@mihnita this is generated, so it'll come out in the wash later

Reply:
Ack.

I suspect the generation will bring these scripts back (and do it with the correct comment)
This is here only to make the tests pass.